src/Database/Database.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace ArmadilloEstates\Database;
+
+
+abstract class Database
+{
+    public static function connect(): \PDO
+    {
+        $db = new \PDO('mysql:host=DB;dbname=armadilloEstates', 'root', 'password');
```

Comment:
can we set the host, dbname, user and password as class constants? makes them easier to change